packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -105,7 +105,7 @@ export interface ISliderProps extends React.Props<SliderBase> {
   /**
    * Optional mixin for additional props on the thumb button within the slider.
    */
-  buttonProps?: React.HTMLAttributes<HTMLButtonElement>;
+  buttonProps?: React.HTMLAttributes<HTMLDivElement>;
```

Comment:
this does seem like a regression; but I don't quite know what the difference is between these two.